### ISSUE TITLE
es alert trigger only once

### ISSUE THREAD

**Issue by hjybdrs** [grafana/grafana#12434]
What Grafana version are you using?
Grafana v5.2.0-beta2 (commit: f76cafa)

What datasource are you using?
elasticsearch

What OS are you running grafana on?
CentOS 7.6

What did you do?
I create a graph with a alert,the alert rules will trigger once when i click the pause button,and then resume it only.can i set a alert rules and continues execute alert rules ？

What was the expected result?
want to continue execute alert rule after alerting or otherelse

What happened instead?
only trigger once
![alert1](https://user-images.githubusercontent.com/6185339/42021782-177aefe4-7aee-11e8-8073-4490b8db17cd.png)
![alert2](https://user-images.githubusercontent.com/6185339/42021783-17ab4ed2-7aee-11e8-96e1-eb2d744aadfb.png)

**Comment by marefr** [comment]
Duplicate of #12356
